src/cecs277TermProject/RoachAutoRefill.java

```diff
@@ -0,0 +1,19 @@
+package cecs277TermProject;
+
+public class RoachAutoRefill  extends MotelAmenities {
+
+	MotelAmenities autoRefill;
+	 
+	public RoachAutoRefill(MotelAmenities autoRefill) {
+		this.autoRefill = autoRefill;
+		}
+	 @Override
+	 public String getDescription() {
+		return ",Auto Refill";
+	 }

```

Comment:
This should be `toString()`, and because it inherit from `Room.java` instead, it should return `super.toString() + ", Auto Refill"`.